batch/batch/server/server.py

```diff
@@ -75,6 +77,22 @@ def make_logger():
 instance_id = uuid.uuid4().hex
 log.info(f'instance_id = {instance_id}')
 
+app = web.Application()
+routes = web.RouteTableDef()
+aiohttp_jinja2.setup(app, loader=jinja2.PackageLoader('batch', 'templates'))
+
+
+def abort(code, reason=None):
+    if code == 400:
+        raise web.HTTPBadRequest(reason=reason)
+    elif code == 404:
+        raise web.HTTPNotFound(reason=reason)
+    raise web.HTTPException(reason=reason)
+
+
+def jsonify(data):
+    return web.json_response(data)
```

Comment:
This seems to call `json.dumps()`

If that's the case, do we need to also call `job.to_json()` when using this method?

Reply:
I am not sure what you mean. This is how the docs said to do it.

Reply:
I thought that when we called jsonify with a value returned from a method `to_json()` we were calling two json serialization functions (since `web.json_response` calls `json.dumps`): 

```
return jsonify([job.to_json() for _, job in job_id_job.items()])
```

However, it appears that's not the case. `job.to_json()` doesn't actually do anything related to json. That should be renamed to `job.to_dict()` or `job.to_object()` I think

Reply:
Our `to_json` should be renamed `to_dict`. A PR that makes that move would be very welcome.

Reply:
I think I did that in this PR, but I'll make a separate PR with those changes.